BSPConvert.Lib/Source/MaterialConverter.cs

```diff
@@ -233,9 +238,23 @@ private void AppendShaderParameters(StringBuilder sb, Shader shader)
 				if (textureStage.rgbGen.HasFlag(ColorGen.CGEN_CONST))
 				{
 					var color = textureStage.constantColor;
-					var colorStr = $"{color[0]} {color[1]} {color[2]}";
-					sb.AppendLine("\t$color \"{" + colorStr + "}\"");
+					var r = Math.Round(255 * Math.Pow((float)color[0] / 255, 2.2), 2); // gamma correct rgb values
```

Comment:
This should be moved/refactored into a reusable function if it isn't already.

Reply:
it does exist in ColorUtil, should I just make that function public?

Reply:
Yeah, `public static`